lib/symbol-provider.coffee

```diff
@@ -19,6 +19,7 @@ class SymbolProvider
   buffer: null
   changeUpdateDelay: 300
 
+  textEditorSelectors: new Set(['atom-pane > .item-views > atom-text-editor'])
   selector: '*'
```

Comment:
> What's the difference between `selector` and `textEditorSelector`? Is selector just for grammars? It's too bad there's some ambiguity now. See any way to unify them or make this clearer?

Right, there is a selector for editors and for grammars. I would like to rename `selector` to make it clearer but at the very least I have to support both for backwards compatibility with the Provider API. Would you like me to start renaming it?